Documentation/Upgrade/ceph-upgrade.md

```diff
@@ -24,14 +24,13 @@ until all the daemons have been updated.
 
 ## Supported Versions
 
-Rook v1.12 supports the following Ceph versions:
+Rook v1.13 supports the following Ceph versions:
 
 * Ceph Reef v18.2.0 or newer
 * Ceph Quincy v17.2.0 or newer
-* Ceph Pacific v16.2.7 or newer
 
-Support for Ceph Pacific (16.2.x) will be removed in the next Rook release. It will be mandatory to
-upgrade to Quincy or Reef before upgrading to the Rook release after v1.12.x.
+Support for Ceph Pacific (16.2.x) is removed in Rook v1.13. Upgrade to Quincy or Reef before upgrading
+to Rook v1.13.
```

Comment:
I wonder if this should be a `!!! warn` to make it more attention grabbing?

Reply:
Agreed, we will need to make it obvious in the release notes, release blog, warnings in the breaking changes, etc. But I'd suggest we evaluate the warning when we do a more thorough review of the upgrade guide.